File: apps/pos/native-android/app/src/main/java/com/desktopkitchen/pos/models/Branding.kt
package com.desktopkitchen.pos.models

import com.squareup.moshi.JsonClass

@JsonClass(generateAdapter = true)
data class TenantBranding(
    val primaryColor: String? = null,
    val logoUrl: String? = null,
    val restaurantName: String? = null
)

File: apps/pos/native-windows/DesktopKitchenPOS/Models/Branding.cs
using System.Text.Json.Serialization;

namespace DesktopKitchenPOS.Models;

public record TenantBranding
{
    [JsonPropertyName("primaryColor")]
    public string? PrimaryColor { get; init; }

    [JsonPropertyName("logoUrl")]
    public string? LogoUrl { get; init; }

    [JsonPropertyName("restaurantName")]
    public string? RestaurantName { get; init; }
}
